knotx-performance-tests/jmeter-testplan/knotx.jmx

```diff
@@ -0,0 +1,1517 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We probably should consider adding [Apache License Header](https://github.com/Cognifide/knotx/blob/master/LICENSE#L189) in all `XML` files.
You may take example for XML from [pom.xml](https://github.com/Cognifide/knotx/blob/master/pom.xml#L2).